commands/volumes/volume-status.go

```diff
@@ -69,8 +77,8 @@ func checkStatus(ctx transaction.TxnCtx) error {
 			}
 		}
 
-		brickStatus := &brick.Brickstatus{
-			BInfo:  binfo,
+		brickStatus := &brickstatus{
```

Comment:
Shouldn't you be using `api.BrickStatus` instead of a this local struct?

Reply:
This is is an internal struct used only during the scope of the transaction.

Reply:
Why though? Both the structs have the same members.

Reply:
Okay. I see that the Info members are different, (api.Brickinfo vs brick.Brickinfo). I'm approving the PR.